Formula/o/odin.rb

```diff
@@ -81,6 +81,32 @@ def install
           "-I#{Formula["raylib"].include}", "-L#{Formula["raylib"].lib}", "-lraylib"
         cp "libraygui.dylib", buildpath/raylib_installpath
       end
+    else
+      raylib_installpath = "vendor/raylib/linux"
+
+      # Create symbolic links for GLFW and Raylib libraries
+      ln_s Formula["glfw"].lib/"libglfw3.a", buildpath/"vendor/glfw/lib/darwin/libglfw3.a"
+      ln_s Formula["raylib"].lib/"libraylib.a", buildpath/raylib_installpath/"libraylib.a"
```

Comment:
These are very similar across Linux and macOS so should be moved out of this `if`-`else` block and made unconditional. (You can define `raylib_installpath` conditionally.)